### PR TITLE
Justin/process scraped data

### PR DESCRIPTION
**Pull request by jtyj-github**
## Description

created utility functions to handle the data obtained from running the web scraper. 
- getJSON fetches the data from the webscraper and stores it in a JSON format for later use
- readLocalJSON parses the JSON file
- writeStore then writes the parsed data into a local file
- readStore reads the data from the local file
- compareList compares the newly read data and compares it to previous existing data, updating it if there is new data

### Screenshots / Videos

ket

### CODE REVIEW

File: src/common/utils/scraper.ts [hunk 1]
```diff
@@ -1,6 +1,6 @@
-import puppeteer, { Browser } from 'puppeteer';
-import prisma from '../../common/lib/prisma';
-import { TotoResult, TotoWinningPool } from '@prisma/client';
+import { Browser } from 'puppeteer';
+import prisma from '../lib/prisma';
+import { TotoResult } from '@prisma/client';
```
Comment (andrehadianto) [inline src/common/utils/scraper.ts:3]: import order seems wrong 🤔 . can we also use absolute import for `prisma`

File: src/common/utils/scraper.ts [hunk 2]
```diff
@@ -61,13 +60,13 @@ export default async function totoScrape(browser: Browser): Promise<TotoResult[]
         return [];
     }
 
-    // Save to prisma
+    console.log({results});
```
Comment (andrehadianto) [inline src/common/utils/scraper.ts]:
```suggestion

```

File: temp/sg_lottery.json [hunk 3]
```diff
@@ -0,0 +1,86 @@
+[
```
Comment (andrehadianto) [inline temp/sg_lottery.json]: do we have to commit this?

File: uploads/topics.json [hunk 4]
```diff
@@ -0,0 +1,36 @@
+[
```
Comment (andrehadianto) [inline uploads/topics.json]: do we have to commit this? maybe just gitignore `temp` and `uploads`

**Review by andrehadianto — CHANGES_REQUESTED** [review]

File: src/common/utils/scraper.ts (hunk 1 above)
Comment (jtyj-github) [reply src/common/utils/scraper.ts:3]: updated to absolute imports

**Review by andrehadianto — APPROVED** [review]
LGTM!!!!

**Pull request merged**